Kk wdac edits (#9567)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/windows-defender-application-control/wdac-and-applocker-overview.md

ms.technology: mde
---

> ## Windows Defender Application Control and AppLocker Overview

Reviewer: This should be a header1 i.e. there needs to be only 1 "#" not 2

> ## Windows Defender Application Control and AppLocker Overview

Reviewer 2: ```suggestion
# Windows Defender Application Control and AppLocker Overview
```

- The [path from which the app or file is launched](select-types-of-rules-to-create.md#more-information-about-filepath-rules) (beginning with Windows 10 version 1903)
- The process that launched the app or binary

Note that prior to Windows 10 version 1709, Windows Defender Application Control was known as configurable code integrity (CCI). WDAC was also one of the features that comprised the now-defunct term "Device Guard."

### WDAC System Requirements

WDAC policies can be created on any client edition of Windows 10 build 1903+, or on Windows Server 2016 and above.

> WDAC policies can be applied to devices running any edition of Windows 10, or Windows Server 2016 and above, via a Mobile Device Management (MDM) solution, e.g. Intune; a management interface, e.g. Configuration Manager; or a script host, e.g. PowerShell. Group Policy can also be used to deploy WDAC policies to Windows 10 Enterprise edition, or Windows Server 2016 and above, but cannot deploy policies to devices running non-Enterprise SKUs of Windows 10.

Reviewer 2: ```suggestion
WDAC policies can be applied to devices running any edition of Windows 10, or Windows Server 2016 and above, via a Mobile Device Management (MDM) solution, for example, Intune; a management interface such as Configuration Manager, or a script host such as PowerShell. Group Policy can also be used to deploy WDAC policies to Windows 10 Enterprise edition, or Windows Server 2016 and above, but cannot deploy policies to devices running non-Enterprise SKUs of Windows 10.
```
